### PR TITLE
Add support for download of files larger than 1 MB.

### PR DESCRIPTION
**Pull request by kjozic**

### CODE REVIEW

File: download.go [hunk 1]
```diff
@@ -219,3 +238,37 @@ func downloadFile(authStr, url, filePath string) error {
...
+	req.Header.Add("Authorization", authStr)
+	req.Header.Add("User-Agent", "go-gl/glow")
+	resp, err := client.Do(req)
+
+	if err != nil {
+		return err
+	}
+	defer resp.Body.Close()
+
+	var blob blobContent
```
Comment (errcw) [inline download.go]: downloadBlob & downloadFile are particularly similar, perhaps we could have a single implementation? fileContent & blobContent could both implement Content()?

**Comment by kjozic** [comment]
Only downloadBlob() is enough because all files can be downloaded as blobs.
The problem occurs in log messages because we don't work with filenames anymore. Now we use SHA sums, so log messages don't have any meaning for the regular user. My suggestion is to print local file names.

File: download.go [hunk 2]
```diff
@@ -51,6 +51,14 @@ type fileContent struct {
 	Links       linkUrls `json:"_links"`
 }
 
+type blobContent struct {
```
Comment (errcw) [inline download.go]: I believe fileContent above is no longer necessary?

**Comment by errcw** [comment]
Thanks again!

**Pull request merged**